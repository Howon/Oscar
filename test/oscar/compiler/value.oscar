def main() => unit = {
  int x = 4;
  int y = 10;
  Println(x);
  Println(x * -5);
  Println(x / -5);
  Println(x - -5);

  Println(x - y);
  Println(x + y);
  Println(x * y);
  Println(x / y);
}