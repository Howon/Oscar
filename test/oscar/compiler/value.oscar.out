4
-20
0
9
-6
14
40
0